mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -837,7 +837,31 @@ def _overwrite_metric(new_metric, old_metric):
         if new_latest_metric_dict:
             self._save_to_db(session=session, objs=list(new_latest_metric_dict.values()))
 
-    def get_metric_history(self, run_id, metric_key):
+    def get_metric_history(self, run_id, metric_key, max_results=None, page_token=None):
+        """
+        Return all logged values for a given metric.
+
+        :param run_id: Unique identifier for run
+        :param metric_key: Metric name within the run
+        :param max_results: An indicator for paginated results. This functionality is not
+            implemented for SQLAlchemyStore.
+        :param page_token: An indicator for paginated results. This functionality is not
+            implemented for SQLAlchemyStore and if the value is overridden with a value other than
+            ``None``, an MlflowException will be thrown.
+
+        :return: A List of :py:class:`mlflow.entities.Metric` entities if ``metric_key`` values
+            have been logged to the ``run_id``, else an empty list.
+        """
+        # NB: The SQLAlchemyStore does not currently support pagination for this API.
+        # Raise if either `max_results` or `page_token` are specified, as the functionality
+        # to support paged queries is not implemented.
+        if page_token is not None:
```

Comment:
This condition doesn't look like `if either `max_results` or `page_token` are specified`.

```suggestion
        if max_results is not None or page_token is not None:
```

Reply:
I forgot to the update the docstring with the PR feedback change. I'll update it.